src/button-icons/buttons-icons.stories.html

```diff
@@ -0,0 +1,83 @@
+<article>
+  <h2>Button icons</h2>
+  <div>
+    <table>
+      <tr>
+        <div>
+          <td></td>
+          <td>
+            Button - Icon
+          </td>
+          <td>
+            Button Icon Filled Neutral
+          </td>
+        </div>
+      </tr>
+      <tr>
+        <div>
+          <td>
+            medium action
+          </td>
+          <td>
+            <button class="omrs-btn-icon-medium">
+              <svg>
+                <use xlink:href="#omrs-icon-search"></use>
+              </svg>
+            </button>
+          </td>
+        </div>
+      </tr>
+      <tr>
+        <div>
+          <td>
+            medium disabled
+          </td>
+          <td>
+            <button class="omrs-btn-icon-medium" disabled>
+              <svg>
+                <use xlink:href="#omrs-icon-search"></use>
+              </svg>
+            </button>
+          </td>
+        </div>
+      </tr>
+      <tr>
+        <div>
+          <td>
+            large action
+          </td>
+          <td>
+            <button class="omrs-btn-icon-large">
+              <svg>
+                <use xlink:href="#omrs-icon-search"></use>
+              </svg>
+            </button>
+          </td>
+        </div>
+      </tr>
+      <tr>
+        <div>
+          <td>
+            large disabled
+          </td>
+          <td>
+            <button class="omrs-btn-icon-large" disabled>
+              <svg>
+                <use xlink:href="#omrs-icon-search"></use>
+              </svg>
+            </button>
+          </td>
+        </div>
+      </tr>
+    </table>
+  </div>
+
+  <div class="stories-example lang-html">
```

Comment:
Would be cool to add knobs that let the user adjust the icon and size with select elements. See the following links for how this could be done. I'd recommend exporting the list of icons from the icons.stories.js file so that it could be reused here.

- https://github.com/openmrs/openmrs-esm-styleguide/blob/master/src/icons/icons.stories.js#L23
- https://github.com/openmrs/openmrs-esm-styleguide/blob/master/src/icons/icons.stories.html#L24

Reply:
I added. Thank you for links.

Reply:
I copied the list of icons because i don't have a pretty way to export that list. Can you suggest one?

Reply:
Yeah see the following links for how to import/export things from one file to another:

- https://github.com/openmrs/openmrs-esm-styleguide/blob/master/src/colors/colors.stories.js#L8
- https://github.com/openmrs/openmrs-esm-styleguide/blob/master/src/icons/icons.stories.js#L7